apps/api.planx.uk/rateLimit.ts

```diff
@@ -38,4 +54,19 @@ const sendEmailLimiter = rateLimit({
     HASURA_ONLY_SEND_EMAIL_TEMPLATES.includes(req.params.template),
 });
 
-export { apiLimiter, sendEmailLimiter };
+/**
+ * Very strict limiter for AI powered endpoints
+ */
+const aiLimiter = rateLimit({
+  message: {
+    error: "TOO_MANY_REQUESTS",
+    message: "[AI limiter]: Too many requests, please try again",
+  },
+  windowMs: TEN_MINUTES_IN_MS,
+  // Higher limit in testing environments
+  max: isTestEnv ? 100 : 10,
```

Comment:
Sense check: 10 requests, per 10 minutes, per IP seems at the low end of reasonable right?

Reply:
aye, probably low end! but good place to start 👌🏼